source/help/index.markdown

```diff
@@ -34,6 +34,8 @@ Have you found an issue in your Home Assistant installation? Please report it. R
 - [Why we can't have the Internet of Nice Things: A home automation primer](https://www.openwest.org/custom/description.php?id=92) at [OpenWest 2017](https://www.openwest.org) - July 2017
 - [Home Automation with Home Assistant](https://github.com/jjmontesl/talk-hass-pydaygalicia2017) at [PyDay Galicia 2017](https://pyday2017.python-vigo.es/gl/) - June 2017
 - [Home Automation with Python](https://www.youtube.com/watch?v=KNFZSSCPUyM) at [GLT 2017](https://glt17.linuxtage.at) - April 2017
+- - [Home Assistant workshop](https://github.com/home-assistant-ecosystem/workshops/tree/main/2017-clt-workshop) at [CLT 2017](https://chemnitzer.linux-tage.de/2017/de/) - March 2017
```

Comment:
`-` instead of `- -`.

see https://deploy-preview-22615--home-assistant-docs.netlify.app/help/